arrow-fx/src/test/kotlin/arrow/fx/MVarTest.kt

```diff
@@ -72,7 +74,7 @@ class MVarTest : UnitSpec() {
           val bb = f2.join().bind()
 
           setOf(aa, bb)
-        }.equalUnderTheLaw(IO.just(setOf(10, 20)), EQ())
+        }.shouldBeEq(IO.just(setOf(10, 20)), EQ())
```

Comment:
Implementation of `equalUnderTheLaw` should change to `shouldBeEq` instead. Much bigger improvement since that'll fix the error message for all currently using `equalUnderTheLaw`.

Reply:
see https://github.com/arrow-kt/arrow-fx/pull/46#discussion_r383931087